pages/docs/features/style-props.mdx

```diff
@@ -574,6 +574,14 @@ import { Button } from "@chakra-ui/react"
   >
   </Box>
 </Box>
+
+// add ::before pseudo element
+<Box
+  _before={{ content: '"🙂"', display: 'inline-block', me: '5px'} // Note: the content value need an extra set of quotes!
```

Comment:
I think you're missing a closing `}` at the end.

Reply:
Also, which prop is `me`? Were you thinking about `mr` as in `marginRight`?

Reply:
me - marginEnd

I'll add the missing `}`

Reply:
Also, can you move the Note comment above `_before`, so the code block doesn't scroll horizontally on desktop?

Reply:
addressed. I force pushed it on the last commit